src/worldmap.ts

```diff
@@ -101,11 +104,31 @@ export default class WorldMap {
     return !_.isEqual(locations, dataPoints);
   }
 
+  // Determines if the log scale is applied and chooses which value to use
+  // Used when calculating circle sizes
+  getValue(dataPoint) {
```

Comment:
Please, please please we are using **Type**Script, not *JavasScript*.

Please specify the input parameter's **type** and the return **type** for the newly introduced methods.